### ISSUE TITLE
no scrolling at all on mobile 

### ISSUE THREAD

**Issue by Duke-Archibald** [poirazis/bb-component-SuperTable#94]
I have a super table connected to an external DB 
on PC I can scroll without any problem 
on mobile I can't, the first X rows (depending on the base size of the widget) are correctly displayed but I can't get more rows

**Comment by 1stthingapps** [comment]
Yep- I have the same issue. Scrolls fine on desktop even with a small window but on mobile no scroll. If I swap out with the native BB table component, keeping everything else the same it works fine so it's specific to the component.
